doc/whats_new/v0.24.rst

```diff
@@ -12,6 +12,22 @@ Version 0.24.2
 Changelog
 ---------
 
+:mod:`sklearn.metrics`
+......................
+
+- |Enhancement| :func:`test_ranking.test_binary_clf_curve_implicit_bytes_pos_label`
```

Comment:
You don't need this entry here. This is not intended to scikit-learn user. We restrain our changelog to public changes.